### PR TITLE
Azure 17.2.0

### PR DESCRIPTION
**Pull request by whites11**
Towards: https://github.com/giantswarm/roadmap/issues/1202

### CODE REVIEW

**Comment by whites11** [comment]
E2e tests green

File: README.md [hunk 1]
```diff
@@ -181,8 +181,13 @@ to all Giant Swarm installations.
     - [v20.0.0-alpha1](https://github.com/giantswarm/releases/tree/master/azure/v20.0.0-alpha1)
 
 - v17
-  - v17.0
+  - v17.2
```
Comment (whites11) [inline README.md:184]: Fixed wrong readme structure in `master`

**Review by T-Kukawka — APPROVED** [review]
Looks good, waiting for any comment from announcement and will release

**Comment by whites11** [comment]
> Looks good, waiting for any comment from announcement and will release

good let me know when I can merge, thanks

**Pull request merged**